### PR TITLE
Tdl 13587 add inventory item data

### PR DESCRIPTION
**Pull request by dbshah1212**
# Description of change
Added the support of Inventory Item

# QA steps
 - [Discover mode, Sync Mode] automated tests passing
 - [Check Discover mode, Sync mode and collect data for Inventory Item] manual qa steps passing (list below)
 
# Risks

# Rollback steps
 - revert this branch

### CODE REVIEW

**Review by karanpanchal-crest — APPROVED** [review]

**Comment by yogeshbhate** [comment]
when would the PR's related to investory item be merged ? is there any ETA ?

File: tests/test_pagination.py [hunk 1]
```diff
@@ -18,13 +18,16 @@ def name(self):
 
 
     def test_run(self):
+        # As it can call for max 100 product_variants and 
+        # we can generate only one inventory_item for one product_variants
+        excepted_streams = {'inventory_items'} 
         with self.subTest(store="store_1"):
             conn_id = self.create_connection(original_credentials=True)
-            self.pagination_test(conn_id, self.store_1_streams)
+            self.pagination_test(conn_id, self.store_1_streams-excepted_streams)
```
Comment (KrisPersonal) [inline tests/test_pagination.py:26]: Have spaces between store_1_streams and - and expected_streams
```suggestion
            self.pagination_test(conn_id, self.store_1_streams - excepted_streams)
```

File: tests/test_pagination.py [hunk 2]
```diff
@@ -18,13 +18,16 @@ def name(self):
...
 
         with self.subTest(store="store_2"):
             conn_id = self.create_connection(original_properties=False, original_credentials=False)
-            self.pagination_test(conn_id, self.store_2_streams)
+            self.pagination_test(conn_id, self.store_2_streams-excepted_streams)
```
Comment (KrisPersonal) [inline tests/test_pagination.py:30]: Include space as suggested above in line 26

**Review by KrisPersonal — CHANGES_REQUESTED** [review]
Overall looks good, but small change suggested

File: tests/test_pagination.py (hunk 2 above)
Comment (dbshah1212) [reply tests/test_pagination.py]: Added

File: tests/test_pagination.py (hunk 1 above)
Comment (dbshah1212) [reply tests/test_pagination.py]: Added

**Pull request merged**